boa/src/builtins/iterable/mod.rs

```diff
@@ -23,9 +25,12 @@ impl IteratorPrototypes {
             array_iterator: ArrayIterator::create_prototype(ctx, iterator_prototype.clone())
                 .as_gc_object()
                 .expect("Array Iterator Prototype is not an object"),
-            string_iterator: StringIterator::create_prototype(ctx, iterator_prototype)
+            string_iterator: StringIterator::create_prototype(ctx, iterator_prototype.clone())
                 .as_gc_object()
                 .expect("String Iterator Prototype is not an object"),
+            map_iterator: MapIterator::create_prototype(ctx, iterator_prototype.clone())
```

Comment:
This should remove the clippy warning:
```suggestion
            map_iterator: MapIterator::create_prototype(ctx, iterator_prototype)
```

Reply:
OK, I see now where that is. Yes it was removed at first because it is not used later so clone is not necessary, but I returned it for consistency. I will remove it.